setup.py

```diff
@@ -8,6 +8,14 @@
     'msgpack>=0.5.0',
 ]
 
+tests_require = [
+    'pytest>=3.4.0',
+]
+
+extras_require = {
+    'pyuv': ['pyuv>=1.0.0'],
```

Comment:
what's this for?

Reply:
I thought this is convention in setuptools to define what your tests requirement is, so that the below works.

python setup.py test